src/current/cockroachcloud/upgrade-policy.md

```diff
@@ -17,7 +17,11 @@ This page describes the upgrade policy for {{ site.data.products.db }}. For self
 
 Patch version [releases](../releases/), or "maintenance" releases, contain stable, backward-compatible improvements to the major versions of CockroachDB (for example, v21.2.0 → v21.2.1).
 
-All {{ site.data.products.serverless }} and {{ site.data.products.dedicated }} clusters are subject to automatic upgrades to the latest supported patch version. **To ensure that connections remain current during these upgrades, it's important to use [connection retry logic](production-checklist.html#keeping-connections-current) in your application.**
+For {{ site.data.products.dedicated }} clusters, [Organization Admins](authorization.html#org-administrator-legacy) can [set a weekly 6-hour maintenance window](cluster-management.html#set-a-maintenance-window) during which available patch upgrades will be applied. Patch upgrades can also be deferred until the next scheduled window. If no maintenance window is configured, {{ site.data.products.dedicated }} clusters will be automatically upgraded to the latest supported patch version as soon as it becomes available.
```

Comment:
> Patch upgrades can also be deferred until the next scheduled window.

Is this referring to the "Delay upgrades" setting? If so that's not accurate (since "delay upgrades" defers upgrading by 60 days).